dlt/common/runners/pool_runner.py

```diff
@@ -170,8 +170,18 @@ def run_pool(
         )
 
     # start pool
-    pool = create_pool(config)
```

Comment:
makes sense! but I'd go even further and make the check below in:
```py
def create_pool(config: PoolRunnerConfiguration) -> Executor:
```
and pick NullExecutor if threading is not available